docs/modules/ROOT/pages/guides/guides-local-multi-node.adoc

```diff
@@ -0,0 +1,106 @@
+= Running a Multi-Node Cluster Locally
+
+== Overview
+Using https://k3d.io[k3d,role=external,window=_blank], a lightweight wrapper to run Rancher’s minimal Kubernetes
+distribution https://github.com/k3s-io/k3s[k3s,role=external,window=_blank], we can run a multi-node cluster within
+Docker for local development. This allows us to verify that we're able to execute particular components on particular
+nodes or node types without necessarily needing access to a cluster with multiple physical nodes. One use case is the
+execution of machine learning inference and training components on nodes with GPU-acceleration available. This
+technique also allows us to verify that scaling is occurring appropriately. For example, we can verify that PySpark
+data delivery pipeline workers are scaling horizontally to improve throughput.
+
+=== Setup
+1. In Rancher Desktop settings, set Container Runtime to `dockerd(moby)`
+2. Install k3d choosing from the following options:
+* `wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash`
+* `curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash`
+* *macOS:* `brew install k3d`
+3. Create multi-node cluster with local dedicated registry, set `agents` argument to desired number of nodes:
+* `k3d cluster create <cluster-name> --agents <num-agents> --registry-create <registry-name>`
+* Example: `k3d cluster create three-node-cluster --agents 3 --registry-create cluster-registry`
+4. Point `kubectl` to your new cluster
+* `kubectl config use-context <cluster-name>`
+* Example: `kubectl config use-context three-node-cluster`
+
+=== Assigning aiSSEMBLE(TM) Components to Particular Nodes or Node Types
```

Comment:
S: The spacing on this section is super janky on the published page. It looks like it is an issue with the code examples and the text not wrapping them properly. Not sure if it can be fixed but if so would be good to do so.
![image](https://github.com/boozallen/aissemble/assets/131482695/e0e42bf0-2e03-41b9-8fae-c3ba674913ed)

Reply:
✅ UPDATED